packages/common/src/content/index.ts

```diff
@@ -1016,3 +1016,22 @@ const getSolutionUrl = (content: IHubContent): string => {
 // instead of hardcoding the types here.
 const isPageContent = (content: IHubContent) =>
   includes(["Hub Page", "Site Page"], content.type);
+
+// proxied csv helpers
+const MAX_SIZE = 5000000;
+export const isProxiedCSV = (item: IItem, requestOptions: IHubRequestOptions) =>
+  !requestOptions.isPortal &&
+  item.access === "public" &&
+  item.type === "CSV" &&
+  item.size <= MAX_SIZE;
+
+export const getProxyUrl = (
+  item: IItem,
+  requestOptions: IHubRequestOptions
+) => {
+  let result;
+  if (isProxiedCSV(item, requestOptions)) {
+    result = `${requestOptions.hubApiUrl}/datasets/${item.id}_0/FeatureServer/0`;
```

Comment:
@tomwayson, it looks like opendata-ui has two divergent ways of defining the hubApiUrl:
- Without `/api/v3` ([hub-service mixin](https://github.com/ArcGIS/opendata-ui/blob/master/packages/ember-arcgis-opendata-components/addon/mixins/hub-service-mixin.js#L81))
- With `/api/v3` ([Layout Card Decorator](https://github.com/ArcGIS/opendata-ui/blob/master/packages/ember-arcgis-opendata-components/addon/decorators/layout-card.js#L16), note the use of `appSettings.hubApiUrl` instead of `appSettings.hubRootDomain`)

It seems like most of the app omits the `/api/v3`, but I wanted to verify the correct url with you.

Reply:
good catch, I know that hub-service-mixin is widely used, and, based on this https://esri.github.io/hub.js/api/common/getHubApiUrl/ I think _without_ is correct, so it looks like this line needs to be updated to append it.

Reply:
@tomwayson opendata-ui uses the pattern on line 1034 to define proxied csv urls (`${siteHostname}/datasets/${item.id}_0/FeatureServer/0`). I don't think we need to change it unless @drspacemanphd thinks otherwise.